packages/rke2-flannel/generated-changes/patch/values.yaml.patch

```diff
@@ -31,7 +31,7 @@
    backend: "vxlan"
    # Port used by the backend 0 means default value (VXLAN: 8472, Wireguard: 51821, UDP: 8285)
 -  #backendPort: 0
-+  backendPort: 0
++  backendPort: 4789
```

Comment:
Do we have to configure the port here? This could introduce issue if you use wireguard as backend